pages/partners/[partner].vue

```diff
@@ -0,0 +1,265 @@
+                  <div class="grid hidden-sm-and-down" :class="getColsCount()">
+                    <!-- copying what's in the current website -->
+                    <v-card flat class="mx-4 xl:mx-3" v-for="product in products">
+                      <v-img
+                        :src="`/static/images/${product.image}`"
+                        :lazy-src="`/static/images/${product.image}`"
+                        class="floating-image"
+                      ></v-img>
+                      <v-card-text
+                        class="text-xs md:text-sm xl:text-md 2xl:text-lg font-Poppins font-weight-light"
+                        style="padding: 25px 5px"
+                      >
+                        <!-- md:text-xs 2xl:text-base -->
+                        {{ product.text }}
+                      </v-card-text>
+                    </v-card>
+                  </div>
+
+                  <!-- PRODUCTS MOBILE VIEW -->
+                  <div class="grid hidden-md-and-up">
+                    <!-- copying what's in the current website -->
+                    <v-carousel cycle hide-delimiters touch show-arrows="hover" progress="#ff7b02">
+                      <v-carousel-item v-for="product in products">
+                        <v-card flat class="mx-4 xl:mx-3">
+                          <v-img
+                            :src="`/static/images/${product.image}`"
+                            :lazy-src="`/static/images/${product.image}`"
+                            class="floating-image"
+                          ></v-img>
+                          <v-card-text class="text-xs font-Poppins" style="padding: 25px 0px">
+                            {{ product.text }}
+                          </v-card-text>
+                        </v-card>
+                      </v-carousel-item>
+                    </v-carousel>
+                  </div>
+                </div>
+              </v-window-item>
+            </v-window>
+          </v-col>
+        </v-row>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script setup>
+import { useRoute } from 'vue-router';
+import { partnersData } from '../../data/partners-data.ts';
+
+const route = useRoute();
+const partnerDetails = ref({});
+const name = ref('');
+const cover = ref('');
+const background = ref('');
+const tagline = ref('');
+const description = ref([]);
+const products = ref([]);
+
+let tab = ref('');
+
+console.log(name.value);
+
+onMounted(() => {
+  partnerDetails.value = partnersData.find((partner) => partner.link === route.params.partner);
+  if (partnerDetails.value) {
+    cover.value = partnerDetails.value.cover;
+    name.value = partnerDetails.value.name;
+    background.value = partnerDetails.value.background;
+    tagline.value = partnerDetails.value.tagline;
+    description.value = partnerDetails.value.description;
+    products.value = partnerDetails.value.products;
+  }
+  useHead({
+    title: name.value,
+  });
```

Comment:
place UseHead inside the if condition.

Reply:
noted, will also implement page not found error

Reply:
kahit gayahin mo nalang muna yung ginawa ko. like this

else {
    throw createError({ statusCode: 404, statusMessage: 'Page not found', fatal: true });
  }

Reply:
yung custom error after nalang magmerge yung na pr ko